src/storage/mod.rs

```diff
@@ -1478,6 +1571,42 @@ impl<E: Engine, L: LockMgr> Storage<E, L> {
             .flatten()
     }
 
+    pub fn batch_async_raw_get(
+        &self,
+        cf: String,
+        gets: Vec<ReadpoolCommand>,
+        callback: BatchCallback<Option<Vec<u8>>>,
+    ) -> impl Future<Item = (), Error = Error> {
+        const CMD: &str = "raw_get";
+        let ctx = gets[0].ctx.clone();
+        let priority = get_priority_tag(ctx.get_priority());
+        let res = self.get_read_pool(priority).spawn_handle(move || {
+            tls_collect_command_count(CMD, priority);
+            Self::with_tls_engine(|engine| {
+                Self::async_snapshot(engine, &ctx)
+                    .and_then(move |snapshot: E::Snap| {
+                        tls_processing_read_observe_duration(CMD, || {
+                            let cf = match Self::rawkv_cf(&cf) {
+                                Ok(x) => x,
+                                Err(e) => return future::err(e),
+                            };
+                            for get in gets {
+                                callback(
```

Comment:
Can we only use one callback for these get requests to reduce the cost?

Reply:
The update roughly reduces 50% gRPC poll CPU and 9% qps in oltp_point_select.